### PR TITLE
Added support for [skip release] flag commit message for Travis CI to skip publishing to Maven Central

### PR DESCRIPTION
**Pull request by adrian-shyshkovskyi-here**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: java
 jdk: openjdk8
 import:
 - source: ".travis/release-flow.yml"
-  if: branch = master and type = push
+  if: branch = master and type = push and commit_message !~ /(\[skip release\])/
```
Comment (dmitriy-abramov) [inline .travis.yml]: Please add description of the flag in CONTRIBUTING.md

Comment (adrian-shyshkovskyi-here) [reply .travis.yml]: Added

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,3 +95,5 @@ Git has the `-s` flag that can sign a commit for you, see example below:
 All opened pull request are being tested by Travis CI before they can be merged to the target branch.
 After the new code is pushed to `master` Travis will run the test suite again, build the artifacts and release them
 to Maven Central repository. The job will automatically increase Artifact Wagon patch version during this process.
+If you don't want your changes to trigger the release - you can add the `[skip release]` flag to your commit message,
```
Comment (dmitriy-abramov) [inline CONTRIBUTING.md]: We recommend doing so in cases when you update CI scripts or documentation like README.md and CONTRIBUTING.md.

Comment (adrian-shyshkovskyi-here) [reply CONTRIBUTING.md]: Added

**Pull request merged**